frontend/src/pages/EventSignInPage/index.tsx

```diff
@@ -41,15 +42,41 @@ function EventSignInPage(): JSX.Element {
               </Grid>
 
               <Grid item>
-                <Typography
-                  className={classes.eventName}
-                  variant='h5'
-                  align='center'
-                >
-                  {event.name}
-                </Typography>
+                <Grid component='div'>
+                  <Typography
+                    className={classes.eventName}
+                    variant={event.name.length < 40 ? ('h3') : ('h4')}
+                    align='center'
+                  >
+                    {event.name}
+                  </Typography>
+
+                  <Typography
+                    className={classes.eventName}
+                    variant='h5'
+                    align='center'
+                  >
+                    {format(parseISO(event.startDate), 'PP')} -{' '}
+                    {format(parseISO(event.startDate), 'p')} to{' '}
+                    {format(parseISO(event.endDate), 'p')}
+                  </Typography>
+                </Grid>
               </Grid>
 
+              <Grid item>
+                <Typography className={classes.eventName}
```

Comment:
Need a `<Grid component>` within the `<Grid item>` here as well to cover the two `<Typography>`

Reply:
Fixed